### PR TITLE
feat(route): 增加对千橡游戏下天书奇谈游戏最新消息的支持

### PR DESCRIPTION
**Pull request by zhkgo**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/imop/tianshu
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [X] New Route / 新的路由
  - [X] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/imop/tianshu.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            target: '/tianshu',
+        },
+    ],
+    name: '全部消息',
+    maintainers: ['zhkgo'],
+    handler,
+};
+
+async function handler(ctx) {
+    const page = ctx.req.param('page') || 1;
```
Comment (TonyRL) [inline lib/routes/imop/tianshu.ts:25]: No page turn. Returning the content from the first page is enough.

Comment (zhkgo) [reply lib/routes/imop/tianshu.ts:25]: yes, the first page is not zero. I changed the example of route.

File: lib/routes/imop/namespace.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import type { Namespace } from '@/types';
+
+export const namespace: Namespace = {
+    name: 'imop',
+    url: 'imop.com',
+
+    zh: {
+        name: '猫扑网',
```
Comment (TonyRL) [inline lib/routes/imop/namespace.ts:8]:
```suggestion
        name: '千橡游戏',
```

**Review by TonyRL — COMMENTED** [review]
https://github.com/DIYgod/RSSHub/pull/16416/files#r1712563407

File: lib/routes/imop/tianshu.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                const $ = load(iconv.decode(response, 'gbk'));
+                item.description = $('.right .right_top .right_bot .articlebox').html();
+                return item;
+            })
+        )
+    );
+
+    return {
+        title: '天书最新消息',
+        link: `${baseUrl}/list/0-${page}.htm`,
```
Comment (TonyRL) [inline lib/routes/imop/tianshu.ts:51]:
```suggestion
    path: '/tianshu',
    categories: ['game'],
    example: '/imop/tianshu',
    radar: [
        {
            source: ['t.imop.com'],
            target: '/tianshu',
        },
    ],
    name: '全部消息',
    maintainers: ['zhkgo'],
    handler,
};

async function handler(ctx) {
    const { data: response } = await got(`${baseUrl}/list/0-1.htm`, { responseType: 'buffer' });
    const $ = load(iconv.decode(response, 'gbk'));
    const list = $('.right .right_top .right_bot .list2 .ul1 ul')
        .toArray()
        .map((item) => {
            item = $(item);
            const href: string = item.find('a').attr('href');
            return {
                title: item.find('a').text(),
                link: href.startsWith('http') ? href : `${baseUrl}${href}`,
            };
        });
    const items = await Promise.all(
        list.map((item) =>
            cache.tryGet(item.link, async () => {
                const { data: response } = await got(item.link, { responseType: 'buffer' });
                const $ = load(iconv.decode(response, 'gbk'));
                item.description = $('.right .right_top .right_bot .articlebox').html();
                return item;
            })
        )
    );

    return {
        title: '天书最新消息',
        link: `${baseUrl}/list/0-1.htm`,
```

**Pull request merged**